addons/web/static/src/scss/bootstrap_review_frontend.scss

```
@@ -219,3 +219,25 @@ $-color-for-gray-200-bg: adjust-color-to-background($body-color, $gray-200);
         }
     }
 }
+
+// Review disabled inputs
+.form-select {
+    // This is necessary to achieve a consistent "border rendering behaviour"
+    // across form-select and form-control, particularly when using semi-transparent borders.
+    background-clip: padding-box;
+
+    &:disabled {
+        background-image: escape-svg(url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'><path fill='none' stroke='#{mix($input-color, $input-bg, 50%)}' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' d='M2 5l6 6 6-6'/></svg>"));
+    }
+}
+
+.form-control {
+    &:disabled, [readonly] {
```

Missing parent selector  😉
```suggestion
    &:disabled, &[readonly] {
```